shared/models/git_repo.rb

```diff
@@ -309,7 +308,6 @@ def commit_changes!(commit_message: nil, file_to_commit: nil, repo_auth: self.re
           logger.debug("No changes in repo #{self.git_config.full_name}, skipping commit #{commit_message}")
         else
           git.commit(commit_message)
-          git.push
```

Comment:
I think we might want to still push, but I'm curious what you think

Reply:
I really deleted this to prevent conflicts over the `ci-repo`. As commented in the other PR #280 should tackle this task and automatically prevent conflicts. I can revert this if necessary.

Reply:
Great, thanks for making that issue. As you suggest, let's tackle it there 🚢